codebundles/aws-c7n-ebs-health/runbook.robot

```diff
@@ -0,0 +1,158 @@
+            ...    reproduce_hint=Review the volume details and usage in the AWS Management Console or CLI.
+            ...    details=${item}        # Include details such as volume ID, size, and region.
+            ...    next_steps="Escalate to service owner to review of unattached AWS EBS volume `${item["VolumeId"]}` in AWS Region `${AWS_REGION}` in AWS account `${AWS_ACCOUNT_ID}`. \n Delete unattached AWS EBS volume `${item["VolumeId"]}` in AWS Region `${AWS_REGION}` in AWS account `${AWS_ACCOUNT_ID}`."
+        END
+    END
+
+
+List Unencrypted EBS Volumes in AWS Region `${AWS_REGION}` in AWS account `${AWS_ACCOUNT_ID}`
+    [Documentation]  Check for unattached EBS volumes in the specified region. 
+    [Tags]    ebs    storage    aws    volume
+    ${c7n_output}=    RW.CLI.Run Cli
+    ...    cmd=custodian run -r ${AWS_REGION} --output-dir ${OUTPUT_DIR}/aws-c7n-ebs-health ${CURDIR}/unencrypted-ebs-volumes.yaml --cache-period 0
+    ...    secret__aws_account_id=${AWS_ACCESS_KEY_ID}
+    ...    secret__aws_secret_access_key=${AWS_SECRET_ACCESS_KEY}
+    ${report_data}=     RW.CLI.Run Cli
+    ...    cmd=cat ${OUTPUT_DIR}/aws-c7n-ebs-health/unencrypted-ebs-volumes/resources.json 
+    RW.Core.Add Pre To Report    ${c7n_output.stdout} 
+
+    ${parsed_results}=    CloudCustodian.Core.Parse EBS Results
+    ...    input_dir=${OUTPUT_DIR}/aws-c7n-ebs-health
+    RW.Core.Add Pre To Report    ${parsed_results}  
+
+    ${clean_output_dir}=    RW.CLI.Run Cli    cmd=rm -rf ${OUTPUT_DIR}/aws-c7n-ebs-health/unencrypted-ebs-volumes
+   
+    TRY
+        ${ebs_volume_list}=    Evaluate    json.loads(r'''${report_data.stdout}''')    json
+        Log    ${report_data.stdout}
+    EXCEPT
+        Log    Failed to load JSON payload, defaulting to empty list.    WARN
+        ${ebs_volume_list}=    Create List
+    END
+
+    IF    len(@{ebs_volume_list}) > 0
+        FOR    ${item}    IN    @{ebs_volume_list}
+            RW.Core.Add Issue        
+            ...    severity=2
+            ...    expected=EBS volumes in AWS Account `${AWS_ACCOUNT_ID}` should be unencrypted or in use
+            ...    actual=EBS volume `${item["VolumeId"]}` in AWS Account `${AWS_ACCOUNT_ID}` is unencrypted
+            ...    title=Unencrypted EBS volume `${item["VolumeId"]}` detected in AWS Account `${AWS_ACCOUNT_ID}`
+            ...    reproduce_hint=Review the volume details and usage in the AWS Management Console or CLI.
+            ...    details=${item}
+            ...    next_steps="Escalate to service owner to review Unencrypted AWS EBS volume `${item["VolumeId"]}` found in AWS Region `${AWS_REGION}` in AWS account `${AWS_ACCOUNT_ID}`. \n Enable encryption of AWS EBS volume `${item["VolumeId"]}` in AWS Region `${AWS_REGION}` in AWS account `${AWS_ACCOUNT_ID}`. \n Delete unencrypted AWS EBS volume `${item["VolumeId"]}` in AWS Region `${AWS_REGION}` in AWS account `${AWS_ACCOUNT_ID}`."
```

Comment:
Next steps should try, as best as possible, to suggest a task title that might exist on the map. In this case, 
`Enable encryption of AWS EBS volume `${item["VolumeId"]}` in AWS Region `${AWS_REGION}` in AWS account `${AWS_ACCOUNT_ID}`.` is not a great next step as it includes the volumeID, which might be too specific if searching for another task on the map. It could be `Enable encryption of AWS EBSs in AWS Region `${AWS_REGION}` in AWS account `${AWS_ACCOUNT_ID}`, with the specific volumeIds in the details field of the issue. In this way, another task might exist that closely matches that next step to automatically enable encryption. 

Also, trim the whitespace following the `\n` delimeters, otherwise they will all start with whitespaces.